scenarios/examples/util.py

```diff
@@ -301,7 +301,7 @@ async def anoncreds_issue_credential_v2(
     )
     holder_indy_record = await holder.event_with_values(
         topic="issue_credential_v2_0_anoncreds"
-        if is_holder_anoncreds
+        if (is_holder_anoncreds or is_issuer_anoncreds)
```

Comment:
This change in `util.py` allows the event `issue_credential_v2_0_anoncreds` from an askar holder (this is what I observed in the test, and also in the alice/faber demo).

Reply:
Tests look great. I do remember this and wasn't exactly sure what to do. I'll need to think about it for a bit.